docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -252,6 +252,8 @@ this table:
     - ``VARBINARY``
   * - ``DATE``
     - ``DATE``
+  * - ``TIMESTAMP_NTZ``
+    - ``TIMESTAMP(6)``
   * - ``TIMESTAMP``
```

Comment:
I see the PR title is about _Support reading `timestamp_ntz`..._
Are there any constraints about writing to `timestamp_ntz` columns from Trino?
Could you please check this, for now, manually ?

In any case, we can't yet add a compatibility test for this feature in the Trino code because we don't test yet against Databricks 13.x / Delta OSS 2.4.x

Reply:
> Are there any constraints about writing to timestamp_ntz columns from Trino?

Our max supported writer version is lower than 7.